region/crateria/central.json

```diff
@@ -3660,12 +3660,75 @@
                   ]
                 }
               ]
+            },
+            {
+              "id": 5,
+              "strats": [
+                {
+                  "name": "G-Mode Morph and Bombs to overload Speed Blocks",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [1],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "Bombs",
+                    {"or": [
+                      {"and": [
+                        "SpringBall",
+                        {"spikeHits": 3}
+                      ]},
+                      {"and": [
+                        "Morph",
+                        {"spikeHits": 1},
+                        {"or": [
```

Comment:
The value of this strat is only for artificial morph, but it could be expanded to go down once we have a way to represent an item not having been picked up. I can now add the ceiling bomb strat, just made a helper for it